api/config.js

```diff
@@ -17,7 +17,7 @@ var config = {
       port: process.env.REDIS_PORT || 6379,
       password: process.env.REDIS_PASSWORD,
       userSessionTrackingEnabled: process.env.ENABLE_USER_SESSION_TRACKING || false,
-      tls: process.env.REDIS_TLS ? { port: process.env.REDIS_TLS_PORT || 6380 } : undefined,
+      ...(process.env.REDIS_TLS === 'true' ? true : false) ? {tls: { port: process.env.REDIS_TLS_PORT || 6380 }} : undefined,
```

Comment:
The change is fine, but the inline ternary is a little wonky. Could be like:

```
tls: process.env.REDIS_TLS === 'true' ? { port: process.env.REDIS_TLS_PORT || 6380 } : undefined,
```

Reply:
thanks, that is what I was thinking needed to be done, somehow i went off in another direction. I will verify that and re-open with changes